assets/src/data/model/attendee/index.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * External imports
+ */
+import { isUndefined } from 'lodash';
+import PropTypes from 'prop-types';
+
+import {
+	getQueryString as baseGetQueryString,
+	QUERY_ORDER_ASC,
+	ALLOWED_ORDER_VALUES,
+} from '../base';
+import * as statusModel from '../status/constants';
+
+const REGISTRATION_STATUS_IDS = [
+	statusModel.STATUS_ID_REGISTRATION_APPROVED,
+	statusModel.STATUS_ID_REGISTRATION_CANCELLED,
+	statusModel.STATUS_ID_REGISTRATION_DECLINED,
+	statusModel.STATUS_ID_REGISTRATION_INCOMPLETE,
+	statusModel.STATUS_ID_REGISTRATION_NOT_APPROVED,
+	statusModel.STATUS_ID_REGISTRATION_PENDING_PAYMENT,
+	statusModel.STATUS_ID_REGISTRATION_WAIT_LIST,
+];
```

Comment:
I know in the other pull the _individual status constants_ are in the status model and that's okay and I think it should stay that way.  However the specific code I'm referring to is this:
```
const REGISTRATION_STATUS_IDS = [
+	'RAP',
+	'RCN',
+	'RDC',
+	'RIC',
+	'RNA',
+	'RPP',
+	'RWL',
+];
```

This is different in that its used to provide an _array_ of registration statuses.   Since its informative about the `registration` model domain, I thought it would be useful if its actually kept with the registration model.  I can totally see other models like Transaction, or Ticket that might build queries relying on a `registration_status` prop in which case it would make sense to pull the specific allowed props enum from the registration model.

So to be clear, this array is acting more as an `enum` definition and thus I think it should reside in the model it relates to.

With that said, if you want to put all the specific status enum-link arrays like this in the status model I'm also fine with that, you could label the constants appropriately in it.

Reply:
ohhhhh ya ok... ya I misinterpreted what you said. ya I think that's a good idea. will fix